lists/ru.csv

```diff
@@ -311,3 +311,34 @@ https://www.svoboda.org/,NEWS,News Media,2018-09-13,OONI,
 https://ru.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by wikipedia based on the most popular language domains per country
 https://ru.wikisource.org/,CULTR,Culture,2019-06-07,Wikimedia,URLs provided by wikipedia based on the most popular language domains per country
 https://ru.m.wikisource.org/,CULTR,Culture,2019-06-07,Wikimedia,URLs provided by wikipedia based on the most popular language domains per country
+https://github.com/russian-suicide/ololo,PUBH,Public Health,2019-08-06,,GitHub government takedown notice from 2014-10-21
+https://github.com/muromec/blog/blob/d0a4b73064b0d2306a6184b75c9bbfeb3daa2a00/repost/suicide.rst,PUBH,Public Health,2019-08-06,,GitHub government takedown notice from 2014-10-21
+https://github.com/numitus/objidlib/blob/master/suicide.txt,PUBH,Public Health,2019-08-06,,GitHub government takedown notice from 2014-10-21
+https://github.com/samm-git/objidlib/blob/master/suicide.txt,PUBH,Public Health,2019-08-06,,GitHub government takedown notice from 2014-10-21
+https://github.com/l29ah/objidlib/blob/master/suicide.txt,PUBH,Public Health,2019-08-06,,GitHub government takedown notice from 2014-10-21
+https://github.com/kleonov/some-text/,PUBH,Public Health,2019-08-06,,GitHub government takedown notice from 2014-10-21
```

Comment:
As these sites are all in HTTPS, to block the sub-page the censor would have to block the entire github.com domain. Maybe we can keep a sample of these that have different domains?

Reply:
Hm true, I totally missed that. But in case of TLS MITM attacks this might still matter, as only the URLs in question might be blocked while other will pass. Nation-wide TLS MITM is not that far fetched: https://news.ycombinator.com/item?id=20472179

Reply:
A TLS MITM would require a custom cert in the user browser. Testing tools should be capable of detecting the TLS MITM before even any request is made.

I see the argument for having one of these to check if they are doing some discrimination between some particular github repos, however I don't think it's necessary to have all of them (test-lists are not block lists).

Reply:
Also true, but there are governments which are trusted CAs and for them this might be easier then.

But you are right, this isn't a block list. Therefore I will adjust the PR accordingly to only contain one URL of the same (sub)domain each.